package.json

```diff
@@ -2,13 +2,15 @@
   "name": "ilp-plugin-lightning",
   "version": "0.1.1",
   "description": "",
-  "main": "build/index.js",
+  "main": "build/src/index.js",
```

Comment:
I would set the `rootDir` in tsconfig so the src files go directly into build. The tests don't need to be in there since ts-jest should handle that, and it's a bit cleaner that way.